sydent/http/servlets/store_invite_servlet.py

```diff
@@ -72,12 +73,20 @@ def render_POST(self, request: Request) -> JsonDict:
             if account.userId != sender:
                 raise MatrixRestError(403, "M_UNAUTHORIZED", "'sender' doesn't match")
 
-        self.sydent.email_sender_ratelimiter.ratelimit(sender)
-
         logger.info(
             "Store invite request from %s to %s, in %s", sender, address, roomId
         )
 
+        try:
+            self.sydent.email_sender_ratelimiter.ratelimit(sender)
+        except LimitExceededException:
```

Comment:
I think it'd be better to make the exception inherit from MatrixRestError so that it automatically does the right thing? https://github.com/matrix-org/sydent/blob/37bd539ed7f9509424e9f25aa5b20bc90ea2da52/sydent/http/servlets/__init__.py#L160-L162

Reply:
Made the change in [acfc89d](https://github.com/matrix-org/sydent/pull/529/commits/acfc89d144e755eea6d7f0675aaff5954854f40e)